### PR TITLE
Add systemd instructions for watch command

### PR DESCRIPTION
**Pull request by agneevX**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -168,6 +168,25 @@ and scrobble plays.
 
 [plex-scrobbler]: https://blog.trakt.tv/plex-scrobbler-52db9b016ead
 
+Create a systemd daemon so that it scrobbles automatically in the background:
```
Comment (glensc) [inline README.md:171]:
```suggestion
Create a systemd unit so that it scrobbles automatically in the background:
```

**Comment by agneevX** [comment]
It seems that `python3 -m plex_trakt_sync watch` isn't correct?

Shouldn't it be `./plex_trakt_sync.sh watch`?

**Comment by glensc** [comment]
Explain your claim!

**Comment by agneevX** [comment]
Ok. The main entrypoint to using PlexTraktSync is the shell script right? So it's only fair that the python executable not be called directly.

**Comment by glensc** [comment]
@agneevX is this ready for merge?

**Comment by agneevX** [comment]
Yes of course.

**Comment by glensc** [comment]
@agneevX why "of course"? you had a question, and you never explained or showed what error you encountered with readme, nor any further feedback.

**Pull request merged**